### DISCUSSION TITLE
Adatbiztonság, kötelezőség

### DISCUSSION THREAD

**Discussion by RabiTibor** [nav-gov-hu/eVAT#4] [Általános]
Saját könyvelőirodai rendszerünk adatbiztonsági okokból úgy épül fel, hogy azon számítógépek, melyeken a könyvelés, bérszámfejtés fut, nem rendelkeznek internet kapcsolattal. Egy külön számítógépet üzemeltetünk arra, hogy azon keresztül, teljesítsük az adatszolgáltatásokat, bevallásokat. Persze ez egy mikro rendszer, nagyban elképzelhetetlen lenne, de úgy gondolom nem vagyunk egyedüliek. Ebben az esetben természetesen nem lesz lehetőség az eÁfa M2M használatára. Részemről ez nem probléma, csak onnantól, ha egyszer mégis kötelezővé kívánják ezt tenni. Ebben az esetben ugyanis jelentős többletkiadással jár a fejlesztés. Belső hálózat kialakítása és biztonsági szoftverek üzemeltetése miatt. Javasolnék egy hosszabb tesztüzemet és lépcsőzetességet a bevont adózók körében.
Jelenleg olyan programmal dolgozunk, ami a könyvelés elkészülte után elkészíti az áfa bevallás xml fájlját, tehát átsorolja a saját rendszeréből a bevallás soraiba az áfát. Gyakorlatból tudom, hogy ez még nem elég, mert a különbözeti áfát alkalmazóknál minden esetben manuálisan kell megadni az adóalapot és az áfát. 
További problémát okoz, hogy alapos lehet a könyvelő, de akkor is téveszthet. Bevallás benyújtása előtt egy hibásan felrögzített gazdasági eseményt bármikor javíthatok, mindenféle bonyodalom nélkül. Ezt egy M2M kapcsolatban nem tudom elképzelni, mert nem lesz olyan egyszerű az adatok megváltoztatása.

**Comment by RuszinZsolt** [comment]
Az adóhatóság úgyis gondolt erre is. 

Ja, nem.

**Comment by vectorkovacspeter** [comment]
Az M2M kapcsolathoz ugyanúgy lehet alkalmazni a köztes gép megoldást, csak arról kell gondoskodni, amiről most az online számla esetében is, hogy a könyvelői gépek belső hálózaton adatcserét tudjanak végezni az Internetképes géppel, ami a tényleges kommunikációt végzi. Régi vágású alkalmazások ezt jellemzően megosztott mappa és időzített alapú megoldással teszik, a modernebbeknek van/lehet erre belső API-juk.

Tévedés kérdése kapcsán szerintem a folyamat lépések leírásában (pdf 10. oldal 2. ábra) benne van, hogy a könyvelő látja leadás előtt, és ő dönt leadja-e, így elvileg nem rosszabb a helyzet ebből a szempontból, mint eddig.

**Comment by renced42** [comment]
Kedves @RabiTibor 

Zárom az issuet. Ha van kérdés nyisd újra.

Köszi!
